_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Global warming may be key to finding E.T.
+author: neuroamanda
+---
+
+_Exoplanets that are too hot may be a sign of alien life_
+
+A new research paper suggests that scanning for waste heat radiating off of Earth-like planets may be important in the search for extraterrestrial intelligent life.
+Like the radio signals emitted by Earth, heat may serve as an interstellar smoke signal indicating energy production and consumption.
+While this not a new idea, it may be a consideration for how we build future large telescopes.
+
+With nearly 2000 exoplanets discovered so far &mdash; of which 40-odd are potentially habitable &mdash; Earth may not be as special in its ability to harbor life as once thought.
```

Comment:
Potentially habitable to Earthlings…

Also, "as once thought" -- how far back are we talking? Because people have long proclaimed that Earth couldn't possibly be special given the vastness of the Universe.